meteor/meteor
# NPM installer should also work with linux/mac

In this PR I change a little the way we were doing before, which was using NPM to link the binary in a "path" included directory.
This won't work for multiplatform, because package.json only allows us to specify one possible file for the bin key.

The strategy is discovering which system we are doing the install, and adding .meteor to the execution PATH, windows included.

## How to test?

- Remove or rename your ~/.meteor folder to ~/.meteor-backup (`mv ~/.meteor ~/.meteor-backup`)
- Verify that the npm "meteor" package is not installed, if it is, uninstall it
- Run `npm link` in this package folder

## Docs

- [X] Update Docs - https://github.com/meteor/docs/pull/748
- [X] Update Guide - https://github.com/meteor/guide/pull/1083
- [X] Update tutorials - https://github.com/meteor/react-tutorial/pull/37 + https://github.com/meteor/blaze-tutorial/pull/27 + https://github.com/meteor/svelte-tutorial/pull/10 + https://github.com/meteor/vue-tutorial/pull/11 + https://github.com/meteor/angular-tutorial/pull/1
- [X] Update Website - **Pending publication.**
